portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Fetches data from java servlet and adds it to DOM tree
+ */
+async function fetchAndInsertDataIntoDOM() {
+	const response = await fetch('/data');
+	const comments = await response.json();
+	const commentsListElement = document.getElementById('comments-list')
+	commentsListElement.innerHTML = '';
```

Comment:
Why is this necessary?

Reply:
It's not really, just a safety check in case there was any hardcoded html in the commentsListElement.